### PR TITLE
Add Amos's guide to build a Rust service

### PR DESCRIPTION
**Pull request by alper**
A lot of Rust but also a very thorough treatment of a lot of parts of Nix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@
 
 ### Learning
 
+* [Building a Rust service with Nix](https://fasterthanli.me/series/building-a-rust-service-with-nix) - About creating a Rust application but treats a lot of Nix as well and is *very* thorough.
```
Comment (cyntheticfox) [inline README.md:45]:
```suggestion
* [Building a Rust service with Nix](https://fasterthanli.me/series/building-a-rust-service-with-nix) - An in-depth blog series about creating a Rust application with Nix.
```

**Review by cyntheticfox — CHANGES_REQUESTED** [review]
Wording could use some grammatical changes, but otherwise looks good.

**Comment by alper** [comment]
Applied suggestions.

**Review by cyntheticfox — APPROVED** [review]
Looks good

**Pull request merged**